js/main.js

```diff
@@ -75,6 +83,8 @@ $ul.addEventListener('click', function (e) {
 
   if (e.target.nodeName === 'I') {
     viewSwap('entry-form');
+    $deleteButton.classList.remove('hidden');
+    $buttonWrapper.style.justifyContent = 'space-between';
```

Comment:
We shouldn't be avoiding inline styling. You should just be adding / removing a class based off of styling you want to apply